README.md

```diff
@@ -427,6 +446,22 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## Frontend Interview Series
```

Comment:
Move this to the Web Development category that you created below

Reply:
Done!